types/filter.d.ts

```diff
@@ -7,8 +7,13 @@ export { FilterType } from "../src/filter";
 
 export declare type SingleValue = string | number | boolean | Date;
 export declare type RangeDataType = number | Date;
+export declare type Param = {
+    value: string;
+    prependTable: boolean;
```

Comment:
>prependTable: boolean; [](start = 4, length = 22)

This feels like it should be CosmosDB specific in the getSingleFilterCosmosDbCondition

Reply:
I need to be able to prepend some but not all parameters to udfs with the chosen table prefix, I don't think I have another option than this. We can sync offline though if you disagree.

---
In reply to: [260920852](https://github.com/ssivov/filter-conditions/pull/14#discussion_r260920852) [](ancestors = 260920852)